app/src/main/java/com/example/metachef/Fragments/ProfileFragment.java

```diff
@@ -67,9 +69,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        btnFav = view.findViewById(R.id.btnFav);
-        btnLogout = view.findViewById(R.id.BtnLogout);
+
+        dialog = new Dialog(getContext());
+        dialog.setContentView(R.layout.custom_dialog);
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP){
```

Comment:
what was this for?

Reply:
I do not remember, I removed it after i noticed it.